barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/mega_arithmetization.hpp

```diff
@@ -309,13 +310,18 @@ template <typename FF_> class MegaArith {
             info("");
         }
 
-        size_t get_total_structured_size()
+        size_t get_structured_dyadic_size()
         {
-            size_t total_size = 0;
+            size_t total_size = 1; // start at 1 because the 0th row is unused for selectors for Honk
             for (auto block : this->get()) {
                 total_size += block.get_fixed_size();
             }
-            return total_size;
+
```

Comment:
just an update to allow this method to compute the dyadic size itself since this is how it was always being used